unit-tests/native/src/test/scala/scala/scalanative/meta/LinktimeInfoTest.scala

```diff
@@ -11,6 +12,10 @@ class LinktimeInfoTest {
     assertEquals(LinktimeInfo.debugMode, !LinktimeInfo.releaseMode)
   }
 
+  @Test def testVersion(): Unit = {
+    assertEquals(LinktimeInfo.runtimeVersion, ScalaNativeBuildInfo.version)
```

Comment:
The test assumes that `nir.Versions.current` and `ScalaNativeBuildInfo.version` are always in sync. Is that correct?

Reply:
Yes, it's correct